source/reference/limitations.txt

```diff
@@ -112,8 +112,8 @@ Sharded Clusters
 
 - The shard key cannot be :ref:`refined <shard-key-refine>` while
   syncing.
-- The shard key cannot be modified using :dbcommand:`reshardCollection`
-  during syncing.
+- The :dbcommand:`reshardCollection` operations from the source cluster
```

Comment:
(nit -- minimalism)
Super non blocking. You can omit "The" if you like.

Reply:
Keeping "The" to be consistent with the other bullet items. Thanks!